operations/payments.md

```diff
@@ -401,6 +401,57 @@ Returns all payments in the system, filtered by various parameters. At least one
 | `Payments` | array of [Payment](#payment) | required | The list of filtered payments. |
 | `Cursor` | string | required | Unique identifier of the last and hence oldest payment returned. This can be used in [Limitation](../guidelines/pagination.md#limitation) in a subsequent request to fetch the next batch of payments. |
 
+## Refund payment
+
+Refunds payment.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/payments/refund`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "EnterpriseId": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+    "PaymentId": "f6313945-94c1-4e27-b402-031c2a8c989f",
+    "AccountId": "35d4b117-4e60-44a3-9580-c582117eff98",
+    "Reason": "Sample reason",
+    "ValueToRefund": 110.5,
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `EnterpriseId` | string | optional | Unique identifier of the [Enterprise](enterprises.md#enterprise). Required when using a [Portfolio Access Token](../guidelines/multi-property.md), ignored otherwise. |
+| `PaymentId` | string | required | Unique identifiers of specific [Payment](payments.md#payment). |
+| `AccountId` | string | required | Unique identifier of the account (for example [Customer](customers.md#customer)) the payment belongs to. |
+| `Reason` | string | required | Refund reason. |
+| `ValueToRefund` | decimal | required | Refund amount. If not provided the whole payment will be refunded. |
+
+### Response
+
+```javascript
+{
+    "PaymentId": "f6313945-94c1-4e27-b402-031c2a8c989f",
+    "RefundId": "1d65c488-111a-4719-b3ea-e1a9969c6069",
+    "Type": "CreditCardPayment",
+    "Amount": { 
+        "Currency": "GBP",
+        "Value": 100
+    },
+}
+```
+
+| `PaymentId` | string | required | Unique identifiers of specific [Payment](payments.md#payment). |
```

Comment:
The table header is missing for the response object:
| Property | Type | Contract | Description |
| :-- | :-- | :-- | :-- |